### ISSUE TITLE
Is it possible to add Informix support?

### ISSUE THREAD

**Issue by BraveOtter** [HeidiSQL/HeidiSQL#783]
## Current behavior

Currently, HeidiSQL is not compatible with Informix databases.

## Expected behavior

I think it would be great to add support for Informix. It is currently a database service widely used in companies. I don't know how difficult it would be to do. It is only a proposal.

**Comment by ansgarbecker** [comment]
According to https://db-engines.com/en/ranking, Informix has a rank of 26, which is not bad, but does not say it belongs to the very popular ones.

Apart from that, I cannot say much here. Implementing a new connection model requires a driver dll with some C or Delphi API and a good documentation on that. And of course a lot of red wine.

**Comment by leeoniya** [comment]
SQLite has almost 5x the score of Informix and an old #12 feature request.

one can always dream :D

**Comment by BraveOtter** [comment]
Yes I know I know. It is not the most used database. Hahahah. it's just the one I use in my work. And now I'm forced to use DBeaver (And I don't like it).

Well, it's just a suggestion.

**Comment by iiugpal** [comment]
Hello BraveOtter,

I had the same request because I use it at work too.
I love informix but it isn't very popular. For a long time, IBM struggled to position its databases in its marketing strategy and i think both DB2 and Informix suffered that internal concurrency. 
I feel Informix is now focused on being an embedded database (for IoT in IBM Cloud Pak' strategy) with NoSQL/JSON, time-series, etc. as legacy information systems based on 4GL tends to disappear.

I hope it doesn't bother the Heidi team if I give you a couple of suggestions here:
1) Try SQuirreL SQL Client (GNU Lesser General Public License). Personally, I enjoy it more than DBeaver to query data
2) Try AGS Server Studio. It is not free, but it is amazing to manage IDS servers

Kr
